seeds/_sector/dex/uniswap_ethereum_base_trades_seed.csv

```diff
@@ -0,0 +1,10 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_bought_amount,token_sold_address,token_sold_amount,token_bought_amount_raw,token_sold_amount_raw
+ethereum,uniswap,1,2022-08-10,0xc5d051f200779582390fe36838263f346e3f9c13731f6d186f12bd72e1223501,164,0x6b175474e89094c44da98b954eedeac495271d0f,114.525972456783,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.062801179402571,114525972456783306496,62801179402571044
+ethereum,uniswap,1,2022-08-10,0x6a6aaf4e55c3f7a070b1b2bb955c8a072bd7f3a00db898bb6ad9c35ff1d0bd36,58,0x23b608675a2b2fb1890d3abbd85c5775c51691d5,0.00485937701376674,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.0941477783366224,4859377013766739,94147778336622368
+ethereum,uniswap,1,2022-08-10,0x73e6503108c7774fad6f297303553153bb074d7a8abea64e67c23025dbd9cdd1,259,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,1604.708387,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.9605157,1604708387,960515700000000000
+ethereum,uniswap,2,2022-08-14,0x644857941ede03358cbb28793d925e374184d07faa62cba22b1ca974035c06ee,161,0x6b175474e89094c44da98b954eedeac495271d0f,149.606421160013,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,150,149606421160013429331,150000000
+ethereum,uniswap,2,2022-08-14,0x51b191c8c620b1dc5867f1c0e1c3b2c4acb20009a7e29a8540c0ed94f678c350,194,0x1f9840a85d5af5bf1d1762f925bdaddc4201f984,1980.88020260497,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,9.13345856353334,1980880202604972474507,9133458563533342720
+ethereum,uniswap,2,2022-08-14,0x77bdb8bff66248595b99851084a3a921db653ba2fb2a775ac7f0614cf4bddab5,23,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,3.13666561702156,0x1f9840a85d5af5bf1d1762f925bdaddc4201f984,683.8,3136665617021563880,683800000000000000000
+ethereum,uniswap,3,2022-09-28,0xdcaf9394fddb84afbf971045661d7e96ed2d01a743ea913abceb20cf4c13452e,2,0xdac17f958d2ee523a2206206994597c13d831ec7,18410.19214,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,13.7977443609023,18410192140,13797744360902256640
+ethereum,uniswap,3,2022-09-28,0x4bda429e77bdd899139e00628c8fee2e119db6098745cba00e11efeeb1a99bd2,84,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,10747.932591,0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,8.06043203550863,10747932591,8060432035508632724
```

Comment:
Small diff in Diff in uniswap_v3 token_sold_amount_raw:
It may be caused by the loss in data-type conversion during decoding.
https://dune.com/queries/2443799?d=1